dockerfiles/bash/Dockerfile

```diff
@@ -6,20 +6,20 @@ RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-ada
 FROM mongo:3.7.9
 
 RUN apt-get update \
-  && apt-get install --no-install-recommends -y curl=7.47.0-1ubuntu2.8 \
+  && apt-get install --no-install-recommends -y curl \
   && rm -rf /var/lib/apt/lists/*
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
 RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
 RUN apt-get update \
   && apt-get install --no-install-recommends -y \
-    nodejs=8.11.3-1nodesource1 \
-    man-db=2.7.5-1 \
-    netcat=1.10-41 \
-    python=2.7.12-1~16.04 \
-    python-pip=8.1.1-2ubuntu0.4 \
-    git=1:2.7.4-0ubuntu1.4 \
-    yarn=1.7.0-1 \
+    nodejs \
+    man-db \
+    netcat \
+    python \
+    python-pip \
+    git \
+    yarn \
```

Comment:
I trust packages like `curl`, but I don't really trust JavaScript packages. If yarn is needed I would prefer to install it with npm so it is pinned. But can't we just use npm in the first place since we install it anyway?

Reply:
This is using the official installation method recommended by yarn in ubuntu/debian. Note above we are adding yarnpkg as a apt source. I suppose we could pin this, but I don't think it makes sense. Debian/ubuntu users trust the updates from apt, since it isn't a rolling forward distribution but rather a stable one. So generally you won't get major version changes, just patches.

Reply:
I'm comfortable unpinning yarn because it's only used to [install packages in the bash language server](https://github.com/mads-hartmann/bash-language-server/blob/8790f7428e42b6c421e2dd224c7b45285809768e/Makefile) and to add a package globally later in this file.